.env.development.local

```diff
@@ -0,0 +1,2 @@
+REACT_APP_USE_EXAMPLE_CUSTOM_ATTRIBUTES=true
+REACT_APP_GOOGLE_MAPS_API_KEY=AIzaSyDZhsJGsRv0c9HykUHfnVAIVOb1tWtqrTg
```

Comment:
This file is in wrong place. 
You should write these variables to the file called `.env`